internal/docs/exported_fields.go

```diff
@@ -46,13 +54,37 @@ func renderExportedFields(fieldsParentDir string) (string, error) {
 		builder.WriteString("(no fields available)\n")
 		return builder.String(), nil
 	}
-	renderFieldsTable(&builder, collected)
+
+	packageRoot, found, err := packages.FindPackageRoot()
```

Comment:
Wdyt about passing the spec version from `generateReadme`? This way we don't need to find the package root in different places and this method keeps depending only on the fields dir.

Reply:
Makes sense. Adapted the code